docs/root/configuration/http/http_filters/compressor_filter.rst

```diff
@@ -0,0 +1,80 @@
+.. _config_http_filters_compressor:
+
+Compressor
+==========
+Compressor is an HTTP filter which enables Envoy to compress dispatched data
```

Comment:
Need links back and forth to relate to existing gzip filter, when one should be used vs. the other.